rig/machine_control/packets.py

```diff
@@ -11,18 +11,46 @@ class SDPPacket(object):
     __slots__ = ["reply_expected", "tag", "dest_port", "dest_cpu", "src_port",
                  "src_cpu", "dest_x", "dest_y", "src_x", "src_y", "data"]
 
-    def __init__(self, reply_expected=False, tag=None,
-                 dest_port=None, dest_cpu=None, src_port=None, src_cpu=None,
-                 dest_x=None, dest_y=None, src_x=None, src_y=None, data=b''):
+    def __init__(self, reply_expected=False, tag=0xff,
+                 dest_port=None, dest_cpu=None, src_port=7, src_cpu=31,
+                 dest_x=None, dest_y=None, src_x=0, src_y=0, data=b''):
         """Create a new SDPPacket.
 
         Parameters
         ----------
+        dest_x : int (0-255)
+            x co-ordinate of the chip to which the packet should be sent.
+        dest_y : int (0-255)
+            y co-ordinate of the chip to which the packet should be sent.
+        dest_cpu : int (0-17)
+            Index of the core which should receive the packet.
+        dest_port : int (0-7)
+            Port which should receive the packet (0 is reserved for debugging).
+        data : bytes
+            Data to append to the packet.
         reply_expected : bool
-            True if a reply is expected, otherwise False.
+            True if a response to this packet is expected, if False (the
+            default) no response is expected.
+
+        Other Parameters
```

Comment:
Numpydoc recommends it https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#id6